Tiltfile

```diff
@@ -113,6 +113,25 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+addon_settings_files = []
+
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
```

Comment:
Just to confirm. To enable our test-extension the following has to be added to tilt-settings.yaml:
```
addon_repos: []
enable_addons:
- test-extension
kustomize_substitutions:
  SERVICE_NAMESPACE: 'default'
```
(of course an empty addon_repos is optional)

I think we should change `SERVICE_NAMESPACE` to something more namespaced, e.g. `TEST_EXTENSION_SERVICE_NAMESPACE`. 


Follow-up: We should extend tilt.md to cover the new implementation. (fine for me to do this in a follow-up PR)